### DISCUSSION TITLE
How to update another input field values?

### DISCUSSION THREAD

**Discussion by rahmat-dev** [edmundhung/conform#309] [Q&A]
I've two input field, name and slug. How to update slug value when I change the name without store slug value in useState?

**Comment by edmundhung** [comment]
Looks like you closed it. But in case you still want some guidance, here is my suggestions:

1. Identify if you really need 2 inputs, if the first input decide the value of the second one, it could be simpler deriving it on the server and remove the 2nd input

2. Since there is no HTML way to set the value of another input without JS. setState is fine! The idea of using defaultValue is only to minimise the use of JS in place you don't need.
